docs/reference/cluster/create_enrollment_token.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[create-enrollment-token]]
+=== Create Enrollment Token API
+++++
+<titleabbrev>Create Enrollment Token</titleabbrev>
+++++
+
+Create an enrollment token to enroll a new node to join an existing cluster with security enabled.
+
+[[create-enrollment-token-api-request]]
+==== {api-request-title}
+
+`POST /_cluster/enrollment_token`
+
+[[cluster-create-enrollment-token-api-prereqs]]
+==== {api-prereq-title}
+
+* You must have the `manage_enrollment` <<privileges-list-cluster,cluster privilege>> to use this API.
+[[cluster-create-enrollment-token-api-desc]]
+==== {api-description-title}
+
+The purpose of the create enrollment token API is to generate a token with which user can enroll a new node
```

Comment:
similarly, mention clients here too. 

can enroll a new node to join -> either "can allow a new node to join" or "can allow a new node to enroll"